swatch-producer-azure/src/main/java/com/redhat/swatch/azure/service/AzureMarketplaceService.java

```diff
@@ -60,10 +63,23 @@ public UsageEventOkResponse sendUsageEventToAzureMarketplace(UsageEvent usageEve
             api.submitUsageEvents(
                 azureMarketplaceProperties.getMarketplaceApiVersion(), usageEvent, null, null);
         break;
-      } catch (ApiException | ProcessingException ex) {
-        log.debug(
-            "Exception occurred during azure marketplace api request, likely expected since credentials are tried at random: {}",
-            ex);
+      } catch (ApiException ex) {
+        int status = ex.getResponse().getStatus();
+        if (HTTP_STATUS_FORBIDDEN == status) {
+          log.debug(
+              "Exception occurred during azure marketplace api request, likely expected since credentials are tried at random",
+              ex);
+        } else {
+          response =
+              new UsageEventOkResponse()
+                  .status(
+                      HTTP_STATUS_CONFLICT == status
+                          ? UsageEventStatusEnum.DUPLICATE
+                          : UsageEventStatusEnum.ERROR);
+          break;
```

Comment:
@kflahert I think it makes sense to return this response when Azure returns these two errors, wdyt?